openedx/features/content_tagging/rest_api/v1/views.py

```diff
@@ -0,0 +1,147 @@
+"""
+Tagging Org API Views
+"""
+
+from openedx_tagging.core.tagging.rest_api.v1.views import TaxonomyView
+
+
+from ...api import (
+    create_taxonomy,
+    get_taxonomies,
+    get_taxonomies_for_org,
+)
+from .serializers import TaxonomyOrgListQueryParamsSerializer
+from .filters import UserOrgFilterBackend
+
+
+class TaxonomyOrgView(TaxonomyView):
+    """
+    View to list, create, retrieve, update, or delete Taxonomies.
```

Comment:
This is just the description of `TaxonomyView`. Please explain how/why this subclass changes `TaxonomyView`.

Also, I'm not sure it's a good idea to duplicate the entire docstring of `TaxonomyView` here, because we have to manually keep them in sync now. What do you think about just saying, "refer to TaxonomyView docstring for usage details" ?

Reply:
> This is just the description of `TaxonomyView`. Please explain how/why this subclass changes `TaxonomyView`.

Done f39182d6b92a3fd96762472e8e9429da262d5b01. I also fixed the URL in the docstring.

> Also, I'm not sure it's a good idea to duplicate the entire docstring of `TaxonomyView` here, because we have to manually keep them in sync now. What do you think about just saying, "refer to TaxonomyView docstring for usage details" ?

I didn't like duplicating it here also, but this text is used in the swagger docs of the API. Having all the info available while consuming the API is also important. I'm not sure which is the best balance here.

![image](https://github.com/openedx/edx-platform/assets/849463/822b9248-8375-4007-8062-3defaa66b8f5)

Reply:
I see. Maybe you can [set the docstring programmatically](https://stackoverflow.com/questions/4056983/how-do-i-programmatically-set-the-docstring) so it's not duplicated in the code, but still available in the swagger UI? But if that's too messy, don't worry about it.

Reply:
I liked the solution, but looking at our codebase, I didn't find any use of it. :worried: Maybe it is too much?

Wrong (out sync) documentation is worse than less documentation, so I opted for the "refer to.." as you suggested in the first place.
Commit: 7ab63047cd6d020fc5adb186bb9cae12929735b0

Let me know what you think!

Reply:
This may be the only case in the codebase where we're subclassing and override a REST API endpoint in this way though.... Either way is fine with me. Your "refer to" seems good for now, but can users actually see the other docstring in the swagger UI?

Reply:
> This may be the only case in the codebase where we're subclassing and override a REST API endpoint in this way though.... Either way is fine with me. Your "refer to" seems good for now, but can users actually see the other docstring in the swagger UI?

Unfortunately no. The other API is not exposed here (only on the standalone version of openedx-learning running). A solution would be to create a static API documentation generated from Swagger, hosted on the project documentation page, but it [doesn't seems trivial](https://gist.github.com/oseiskar/dbd51a3727fc96dcf5ed189fca491fb3?permalink_comment_id=3249997).

Reply:
OK, then my preference would be to use the technique I showed you to append the base class docstring to the derived class's docstring, so that it appears in the swagger docs. But any approach is fine, I don't want to block on this or spend much time on it. You can leave in "refer to" or the full copy of the docs, as you feel best.

Reply:
Thank you, @bradenmacdonald! I'll take a look at this tomorrow to use the dynamic docstring.

I will also update the swagger with the API parameters. I'm using only the docstring here. Some decorators actually expose the parameters to the swagger, and we will have an even better result.

Reply:
Hi @bradenmacdonald!

I started digging and found [openededx/api-doc-tools](https://github.com/openedx/api-doc-tools). I did a lot of experiments, but it didn't work well with inheritance. I also tried [drf-yasg](https://github.com/axnsan12/drf-yasg) itself. Still, I needed to put docstring in the methods (not in the view) to work with inheritance, and I ended up creating more confusion than resolving the doc string problem (it is a promising path thou, we should try it in another moment). I learned a lot but already spent too much time on this.

Do you mind reviewing the PR with the docstring in the current state?

Reply:
Sounds good! We've already spent enough time on this, let's leave it there.